src/plugins/unified_search/public/filter_bar/filter_editor/filter_editor.tsx

```diff
@@ -401,7 +401,7 @@ class FilterEditorComponent extends Component<FilterEditorProps, State> {
 
     return (
       <EuiFormRow fullWidth label={strings.getQueryDslLabel()} helpText={helpText}>
-        <CodeEditor
+        <CodeEditor // TODO CodeOwners will need to help with this
```

Comment:
This seems to work fine now! 

<img width="1052" alt="image" src="https://github.com/elastic/kibana/assets/17003240/bd9cec2c-b1ed-4de2-8b3a-c55abe1dcfad">